dev-proxy/presets/m365.json

```diff
@@ -0,0 +1,195 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/dev-proxy/main/schemas/v1.0/rc.schema.json",
+    "plugins": [
+      {
+        "name": "DevToolsPlugin",
+        "enabled": false,
+        "pluginPath": "plugins\\dev-proxy-plugins.dll",
```

Comment:
I think this will break when you try to use it from a folder other than where proxy is installed. Let's prefix it with `~appFolder\\` so that proxy always resolves it to its installation folder and calling this preset will work from any folder.

Reply:
Good catch, I'll update the paths.